pages/api/events.ics.js

```diff
@@ -0,0 +1,72 @@
+import { createEvents } from "ics";
+
+function parseDateString(dateString) {
```

Comment:
This looks to be the same function used in `pages/api/event.ics.js`. It might be worth extracting this function to another file, and then importing it here and in `pages/api/event.ics.js`.

That way if you decide to change it in the future, then you will only need to change it in one place.